tiny_dnn/layers/recurrent_cell_layer.h

```diff
@@ -28,6 +25,7 @@ namespace tiny_dnn {
  * x(t-1)    x(t)
  *
  **/
+// TODO (@prlz77) template might cause serialization errors. To be discussed.
```

Comment:
Hmm, could you help me to write something better? I don't really know how exactly is serialization working in tiny-dnn.